includes/blocks/class-sensei-block-take-course.php

```diff
@@ -74,8 +74,8 @@ public function render_take_course_block( $attributes, $content ): string {
 		} elseif ( ! is_user_logged_in() ) {
 			$html = $this->render_with_login( $content );
 		}
-
-		return ! empty( $html ) ? '<div class="sensei-block-wrapper sensei-cta">' . $html . '</div>' : '';
+		$wrapper_attributes = get_block_wrapper_attributes( array( 'class' => 'sensei-block-wrapper sensei-cta' ) );
+		return ! empty( $html ) ? '<div' . $wrapper_attributes . '>' . $html . '</div>' : '';
```

Comment:
I think there is a missing space after the `<div` which is causing some Course List issues.

![image](https://user-images.githubusercontent.com/1612178/203556794-955be207-4bdd-4982-8908-16254ba02fd4.png)

Reply:
I fixed it https://github.com/Automattic/sensei/pull/6163/files#diff-b981e2bbec3c37011de89cb5a29cb17879558d14f81a44e59e670a90e17f67caR78